internal/db/schema/migrations/oss/postgres/91/03_indexes_grants_query.up.sql

```diff
@@ -0,0 +1,44 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+  -- For each of these tables, swap the ordering of the
+  -- columns in the index for the primary key.
+  -- This helps the grants query that contains
+  -- several where clauses on what is currently the second
+  -- column in these indexes. By swapping the order, this
+  -- will make it more likely that the query planner will
+  -- choose to use the index.
+  -- See: https://www.postgresql.org/docs/current/indexes-multicolumn.html
+
+      alter table auth_oidc_managed_group_member_account
+  drop constraint auth_oidc_managed_group_member_account_pkey;
+      alter table auth_oidc_managed_group_member_account
+  add primary key (member_id, managed_group_id);
```

Comment:
Can these be combined into one `alter table` statement?

Reply:
Sure